std/range/package.d

```diff
@@ -4663,6 +4667,10 @@ private string lockstepMixin(Ranges...)(bool withIndex, bool reverse)
 
    By default $(D StoppingPolicy) is set to $(D StoppingPolicy.shortest).
 
+   Limitations: Compiler cannot currently infer purity, safety, use of garbage
+   collection nor possibility of an exception from iteration of a lockstep.
+   $(LREF zip) can infer the former two due to a different implementation.
+
    See_Also: $(LREF zip)
```

Comment:
Consider changing "nor" to "or" and "former two" to "first two".

Reply:
I'm bad at being clear so I better heed this.